biz.ganttproject.core/src/main/java/biz/ganttproject/core/chart/scene/gantt/TaskTableSceneBuilder.kt

```diff
@@ -19,31 +19,45 @@ along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
 package biz.ganttproject.core.chart.scene.gantt
 
 import biz.ganttproject.core.chart.scene.gantt.TableSceneBuilder.*
+import biz.ganttproject.core.chart.scene.gantt.TableSceneBuilder.Table.*
 import biz.ganttproject.core.chart.canvas.Canvas
 import biz.ganttproject.core.chart.canvas.TextMetrics
 
 class TaskTableSceneBuilder(
   private val input: InputApi
 ) {
-  private val tableSceneBuilder = TableSceneBuilder(Config(input.rowHeight, input.horizontalOffset))
+  private val tableSceneBuilder = TableSceneBuilder(Config(input.rowHeight, input.horizontalOffset, input.textMetrics))
+  private val cols = listOf(Column("Name"), Column("Begin date"), Column("End date"), Column("Cost"))
```

Comment:
Is this list fixed just for testing? In reality it needs to be configurable, of course.